vyper/semantics/analysis/module.py

```diff
@@ -485,13 +485,13 @@ def _load_import_helper(
 
 
 def _parse_and_fold_ast(file: FileInput) -> vy_ast.VyperNode:
-    ret = vy_ast.parse_to_ast(
+    ast = vy_ast.parse_to_ast(
```

Comment:
i guess since folding isn't performed any longer we can kind of skip the intermediate variable assignment (and rename the function!)

Reply:
i just reverted the change since it's not relevant to this PR